### ISSUE TITLE
Constants break standard highlighting when passed into a function in the following format.

### ISSUE THREAD

**Issue by mahyarmirrashed** [redhat-developer/vscode-java#1441]
##### Environment
- Operating System: Windows 10
- JDK version: openjdk 11.0.6 2020-01-14, OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.6+10), OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.6+10, mixed mode)
- Visual Studio Code version: 1.450 (user setup)
- Java extension version: 0.61.0

##### Steps To Reproduce
1. Format a function like the above and pass constants as arguments. The last argument, only when it is a constant, gets highlighted green for some reason.
2. Be sure to format the method call as I have done and it will always happen. However, when the parenthesis is returned right next to the constant, the error will vanish. The error persists even with native Java methods and my own methods.

##### Current Result

![wrong_highlighting](https://user-images.githubusercontent.com/59240843/81619991-6ec2aa80-93b0-11ea-9aec-b8dda476c88d.png)

##### Expected Result

For the coloring to not switch to green, or preferably, have it consistent among all the constants.

**Comment by fbricon** [comment]
I can reproduce if semantic highlighting is disabled. When enabled, it looks fine. Please try setting
`"java.semanticHighlighting.enabled": true`

You need to also make sure you have `"editor.semanticHighlighting.enabled": true`

**Comment by mahyarmirrashed** [comment]
So I tried going to my settings.json file and adding the settings as suggested but they do not change, unfortunately. I did some more exploring on my own and it works somewhat weirdly.

For example, changing the third and fourth lines to "UNKNOWN_PARAMETER, UNKNOWN_PARAMETER" as one line will correctly color the first parameter but have incorrect coloring on the second. It seems this only occurs when a constant is followed by a newline character.
